js/techtree.js

```diff
@@ -445,7 +445,7 @@ class Lane {
             for (let i = 0; i < this.rows[r].length; i++) {
                 this.rows[r][i].y = offsets[r];
                 this.rows[r][i].x = x;
-                this.rows[r][i].width = element_height;
+                this.rows[r][i].width = element_height; // Todo: check warning `lane_width`?
```

Comment:
Todo: check warning `lane_width`

Reply:
It's a square :upside_down_face:

Reply:
IDE showed warning. Was not sure so I wrote it down.

Reply:
Renamed it to `element_length` :smile: